cfgov/unprocessed/css/molecules/related-topics.less

```diff
@@ -0,0 +1,81 @@
+/* topdoc
+  name: Related Topics
+  family: cfgov-molecules
+  patterns:
+    - name: Default example
+      markup: |
+    <div class="m-related-topics">
+        <h2 class="header-slug">
+            <span class="header-slug_inner">
+                Related Topics
+            </span>
+        </h2>
+        <ul class="list list__unstyled list__links">
+            <li class="list_item">
+                <a href="" class="list_link">
+                </a>
+            </li>
+        </ul>
+    </div>
+    codenotes:
+      - |
+        Structural cheat sheet:
+        -----------------------
+        .m-related-topics
+          .header-slug
+            .header-slug_inner
+        .list.list__unstyled.list__links
+          .list-item
+            .list-link
+  tags:
+    - cfgov-molecules
+*/
+
+@bullet-font-size__em: unit(15px / @base-font-size-px, em);
+
+.m-related-topics {
+
+    .list_link {
+        .h6();
+
+        position: relative;
+        display: inline-block;
+        margin: 0 0 0 @bullet-font-size__em;
+        text-transform: uppercase;
+
+        .u-link__colors(@gray, @gray, @gray, @gray, @gray,
+                        @gold-80, @gold-80, @gold-80, @gold-80, @gold-80);
+
+        &:before {
+            position: absolute;
+            left: -@bullet-font-size__em;
+            width: @bullet-font-size__em;
+            content: "\2022";
+            color: @gold-80;
+        }
+    }
+
+    .list_item {
+        margin-bottom: unit(15px / @base-font-size-px, em);
```

Comment:
fixed in https://github.com/cfpb/cfgov-refresh/commit/127e98d41fd908bc46a3b3860431e3edc2edd179